variables.tf

```diff
@@ -0,0 +1,9 @@
+variable area_code{
+    type=string
+}
+
+variable vpc_cidr {}
+
+variable private_cidr {}
+
+variable public_cidr {}
```

Comment:
Please always make sure all your files have a trailing newline.

A trailing newline is a newline character (\n) placed at the end of a text file or a string. It is essential to add it because many programming languages and tools expect the last line of a file to end with a newline character.

In Terraform, It's not a hard requirement but rather good coding style practice. Having said that, it's still important.